ucm2/USB-Audio/Focusrite/Scarlett-2i4-gen2-HiFi.conf

```diff
@@ -0,0 +1,140 @@
+Include.pcm_split.File "/common/pcm/split.conf"
+
+Macro [
+	{
+		SplitPCM {
+			Name "scarlett2i4_stereo_out"
+			Direction Playback
+			Format S32_LE
+			Channels 2
+			HWChannels 4
+			HWChannelPos0 FL
+			HWChannelPos1 FR
+			HWChannelPos2 FL
+			HWChannelPos3 FR
+		}
+	}
+	{
+		SplitPCM {
+			Name "scarlett2i4_mono_in"
+			Direction Capture
+			Format S32_LE
+			Channels 1
+			HWChannels 2
+			HWChannelPos0 MONO
+			HWChannelPos1 MONO
+		}
+	}
+]
+
+Define.PCMCTLName "PCM Playback"
```

Comment:
It's not required to use a variable for the control name here. Use the value directly.

Reply:
thanks! Fixed.